packages/kit/src/runtime/server/index.js

```diff
@@ -195,6 +195,13 @@ export async function respond(request, options, state = {}) {
 				throw new Error('request in handle has been replaced with event' + details);
 			}
 		});
+
+		// TODO remove for 1.0
```

Comment:
Validation might still be useful after 1.0 to make sure users don't do anything stupid. Personally I would remove the todo

Reply:
> Validation might still be useful after 1.0 to make sure users don't do anything stupid. Personally I would remove the todo

I'm obviously not on the Svelte developer team, but I agree with that, the check makes sense before and after 1.0 version

Reply:
Checking might be useful, but pointing to the PR is specifically to help people migrate. After 1.0 they should just be directed to the docs. I changed the comment instead of removing it